src/StructuralSolvers/Solutions.jl

```diff
@@ -55,7 +55,7 @@ solver(sol::AbstractSolution) = sol.solver
 Solution that stores all intermediate arrays during the analysis.
 """
 struct Solution{ST<:AbstractStaticState,A<:AbstractStructuralAnalysis,
-                SS<:Union{AbstractSolver,Nothing}} <: AbstractSolution
+                SS<:Union{AbstractSolver,SciMLBase.AbstractLinearAlgorithm}} <: AbstractSolution
```

Comment:
This is removed for the case of `LinearStaticAnalysis` where only SciML linear algo is used. Instead of `Nothing` which is our internal representation for `Solver` we should point to the solver used.

Reply:
ok